src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -69,6 +69,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PrepareResourcesDependsOn>
   </PropertyGroup>
 
+  <!-- List of targets that emit Dependency DiagnosticMessages items -->
+  <PropertyGroup>
+    <EmitsDependencyDiagnosticMessages>
+      RunResolvePackageDependencies;
+      $(EmitsDependencyDiagnosticMessages)
+    </EmitsDependencyDiagnosticMessages>
+  </PropertyGroup>
```

Comment:
Generally the property for depends on is XxxDependsOn, but we've also moved away from that in favor of BeforeTargets. I wonder if we shouldn't just have things use BeforeTargets=ResolvePackageDependencies and skip having this list? What else do we think will go here?